video/video_decoder.h

```diff
@@ -397,6 +387,17 @@ class VideoDecoder {
 	 */
 	bool setDitheringPalette(const byte *palette);
 
+	/**
+	 * Set the default high color format for videos that convert from YUV.
+	 *
+	 * This should be called after loadStream(), but before a decodeNextFrame()
+	 * call. This is enforced.
+	 *
+	 * @param palette The palette to use for dithering
```

Comment:
Seems you forgot to change the doxygen parameter description here

Reply:
OK, done.